policyengine_canada/variables/gov/provinces/qc/tax/income/credits/sa/qc_sa_single_eligible.py

```diff
@@ -0,0 +1,19 @@
+from policyengine_canada.model_api import *
+
+
+class qc_sa_single(Variable):
+    value_type = bool
+    entity = Household
+    label = "Quebec senior assistance tax credits eligible senior did not have a spouse"
+    definition_period = YEAR
+    defined_for = ProvinceCode.QC
+
+    def formula(household, period, parameters):
+        person = household.members
+
+        p = parameters(period).gov.provinces.qc.tax.income.credits.sa
+
+        age_eligible = person("age", period) >= p.age_eligibility
+        if_spouse = person("is_spouse", period)
```

Comment:
For consistency with the other file. Could do `~person("is_spouse", period)` directly below
```suggestion
        spouse = person("is_spouse", period)
```